src/app/admin/pontuacao/edit-pontuacao/edit-pontuacao.component.ts

```diff
@@ -15,11 +19,15 @@ export class EditPontuacaoComponent implements OnInit {
   formPontuacao: FormGroup;
   pontuacaoObs: Observable<any>;
   querying: boolean = false;
+  delete;
```

Comment:
e pode fazer o mesmo aqui 
```suggestion
  delete: any = [];
```